src/hooks/useDispatch.ts

```diff
@@ -1,7 +1,9 @@
 // eslint-disable-next-line no-restricted-imports
 import { useDispatch as _useDispatch } from "react-redux";
+import { ThunkDispatch } from "redux-thunk";
 import { RootActions } from "store/actions";
-import { Dispatch as _Dispatch } from "redux";
+import { RootState } from "index";
 
-export const useDispatch = () => _useDispatch<_Dispatch<RootActions>>();
+export const useDispatch = () =>
```

Comment:
Not sure about whether I understand it correctly, but wouldn't it be better to have it like this:

```
export const useDispatch = _useDispatch<ThunkDispatch<RootState, void, RootActions>>;
```